### ISSUE TITLE
InfiniteLoader when scroll reverse do not respect minimumBatchSize

### ISSUE THREAD

**Issue by rodrigocorsi2** [bvaughn/react-virtualized#217]
If you test [InfiniteLoader.example.js](https://github.com/bvaughn/react-virtualized/blob/master/source/InfiniteLoader/InfiniteLoader.example.js) with some changes:

change line 26
```javascript
    this.state = {
      loadedRowsCount: 0,
      loadedRowsMap: {},
      loadingRowsCount: 0,
      randomScrollToIndex: 500 //change here
    }
```

add a `console.log` line 122
```javascript
  _loadMoreRows ({ startIndex, stopIndex }) {
    const { loadedRowsMap, loadingRowsCount } = this.state
    const increment = stopIndex - startIndex + 1
    console.log(startIndex+" => "+stopIndex+' increment:'+increment); //add this line
    for (var i = startIndex; i <= stopIndex; i++) {
```

when you scroll in reverse direction you can see the increment is always 1.

how can I do `InfiniteLoader` respect `minimumBatchSize` in both directions?

**Comment by bvaughn** [comment]
Looks like a bug.

My hands are pretty full with the [version 7](https://github.com/bvaughn/react-virtualized/pull/206/) update  at the moment so I'm probably not going to get to this for a couple of days.

**Comment by bvaughn** [comment]
@rodrigocorsi2: This has been fixed in version 7.0.5. Thanks for the bug report. :)

**Comment by rodrigocorsi2** [comment]
Thank you @bvaughn !
